screens/setting.js

```diff
@@ -0,0 +1,12 @@
+import React, { Component } from 'react';
+import { StyleSheet, Text, View, Image } from 'react-native';
+
+export default class Setting extends React.Component {
```

Comment:
fun fact
This doesn't actually have to be a component

```jsx
const Setting = ( ) => (
      <View style={{ flex: 1, justifyContent: 'center', alignItems: 'center' }}>
        <Text>Settings!</Text>
      </View>);

export default Setting;
```
(I apologies if there are any syntax issues)

However, since we're most likely going to states this component, then I agree with this approach.